scripts/health/common.py

```diff
@@ -0,0 +1,40 @@
+""" """
+import pandas as pd
+import requests
+
+
+def query_who(code: str):
+    """Query the WHO website for a given code.
+    To be replaced in bblocks
+    """
+
+    URL = 'https://ghoapi.azureedge.net/api/'
+
+    request = requests.get(URL + code)
+    data = request.json()
+    df = pd.DataFrame.from_records(data['value'])
+
+    return df
+
+
+def get_malaria_data() -> dict:
+    """Extract and clean malaria data for overview chart"""
+
+    malaria_dict = (query_who('MALARIA_EST_DEATHS')
+        .loc[lambda d: d.SpatialDim.isin(['GLOBAL', 'AFR']), ['SpatialDim', 'TimeDim', 'NumericValue']]
+        .sort_values('TimeDim')
+        .groupby(['SpatialDim'], as_index=False)
```

Comment:
TimeDim gets read as an integer since there are no missing data or errors. However since you are sorting by this, it is important to make sure it always gets read as an int - or even better a datetime. 

Two options: `astype()` or where you `pd.DataFrame.from_records()` do a parse dates for that column